OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -33,28 +33,8937 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+                                <msItem xml:id="ms_i1.3.26">
+                                    <locus from="75rb" to="76vb"/>
+                                    <title type="complete" ref="LIT4035SenkessarDL#Teqe25"/>
+
+                                    <msItem xml:id="ms_i1.3.26.1">
+                                        <locus from="75rb" to="75va"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Teqe25Abib"
+                                            >Commemoration of ʾabbā Bulā</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.26.2">
+                                        <locus from="75va" to="76rc"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Teqe25Ablo"
+                                            >Commemoration of ʾabbā Ablo</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.26.3">
+                                        <locus from="76rc" to="76va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe25YulyosChurch"
+                                            >Commemoration of the consecration of the church of
+                                            Yolyos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.26.4">
+                                        <locus target="#76va"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Teqe25Abel"
+                                            >Commemoration of ʾabbā ʾAbel and Dalilā</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.26.5">
+                                        <locus target="#76va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe25SalAbib">Salām-hymn to
+                                            ʾAbib</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.26.6">
+                                        <locus target="#76va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe25SalComAbib">Salām-hymn to
+                                            the companions of ʾAbib</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.3.26.7">
+                                        <locus from="76va" to="76vb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Teqe25SalAbib2">ʿĀrke-hymn on
```

Comment:
When do you write ʿĀrke-hymn "to" and when "on"?

Reply:
In principle, "to" only when the saint is addressed directly. It might be that there are inconsistencies. Here, the beginning of the hymn is ተዘኪረከ፡ ክርስቶስ፡ ሥቃያተ፡ አቢብ፡ ብዙኃ፨. I would say that Christ is addressed and the hymn is about ʾAbib, or rather a supplication of the author, it continues later in the first person, ኢታጽንዕ፡ ብየ፡ ልበከ፡ ርኅርኃ... which I assume refers to the author...

Reply:
I just wanted to point to this difference for consistency